invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -849,32 +851,48 @@
           "items": {"$ref": "#/definitions/agent"}
         },
 
-        "embargo_date": {
-          "description": "Embargo date of record (ISO8601 formatted date time in UTC). At this time both metadata and files will be made public.",
-          "type": "string",
-          "format": "date-time"
-        },
-
-        "access_condition": {
-          "description": "Conditions under which access to files are granted",
+        "embargo": {
+          "description": "Description of the embargo on the record.",
           "type": "object",
+          "required": ["active", "until", "reason"],
           "additionalProperties": false,
           "properties": {
-            "condition": {
+            "active": {
+              "description": "Whether or not the embargo is (still) active.",
+              "type": "boolean"
+            },
+            "until": {
+              "description": "Embargo date of record (ISO8601 formatted date time in UTC). At this time both metadata and files will be made public.",
               "type": "string",
-              "description": "Textual description under which conditions access is granted."
+              "format": "date"
             },
-            "default_link_validity": {
-              "type": "integer",
-              "description": "Number of days"
+            "reason": {
+              "description": "The reason why the record is under embargo.",
+              "type": "string"
             }
           }
         },
 
-        "access_right": {
-          "default": "open",
-          "description": "TODO - Access right for record. - should be computed or moved to metadata?",
-          "type": "string"
+        "grants": {
```

Comment:
Comment: For the February release we should not allow the grants to be submitted via the REST API, as we're only implementing the minimal access control.